src/frontend/src/pages/ChangeRequestDetailPage/ReviewChangeRequestView.tsx

```diff
@@ -64,8 +64,7 @@ const ReviewChangeRequestsView: React.FC<ReviewChangeRequestViewProps> = ({
     display: 'block'
   };
 
-  if (cr.type === 'ISSUE' || cr.type === 'DEFINITION_CHANGE' || cr.type === 'OTHER') {
-    const issueCR = cr as StandardChangeRequest;
+  const renderProposedSolutionModal: any = (scr: StandardChangeRequest) => {
```

Comment:
You shouldn't use any, the type here should be something like JSX.Element.

Reply:
there's a good chance you wouldn't need to put the type at all i think

Reply:
typing makes me want to claw my eyes out fr